src/Bootstrappers/MailTenancyBootstrapper.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Stancl\Tenancy\Bootstrappers;
+
+use Illuminate\Config\Repository;
+use Illuminate\Support\Arr;
+use Stancl\Tenancy\Contracts\TenancyBootstrapper;
+use Stancl\Tenancy\Contracts\Tenant;
+
+class MailTenancyBootstrapper implements TenancyBootstrapper
+{
+    protected array $originalConfig = [];
+
+    /**
+     * Tenant properties to be mapped to config (similarly to the TenantConfig feature).
+     *
+     * For example:
+     * [
+     *     'property' => 'config.key.name',
+     * ]
+     */
+    public static array $credentialsMap = [
+        'smtp_transport' => 'mail.mailers.smtp.transport',
+        'smtp_host' => 'mail.mailers.smtp.host',
+        'smtp_port' => 'mail.mailers.smtp.port',
+        'smtp_encryption' => 'mail.mailers.smtp.encryption',
+        'smtp_username' => 'mail.mailers.smtp.username',
+        'smtp_password' => 'mail.mailers.smtp.password',
+        'smtp_timeout' => 'mail.mailers.smtp.timeout',
+        'smtp_local_domain' => 'mail.mailers.smtp.local_domain',
```

Comment:
Wouldn't it make more sense to have these in the opposite direction? People may want to customize the *tenant attribute* used for a certain config key, rather than vice versa.

Reply:
That makes sense, but it could also confuse people who already know TenantConfig, which has similar logic. Though this doesn't exactly copy the TenantConfig logic – it's different, so this might not be an issue

Reply:
There's **tons** of use cases and approaches to changing config in multi-tenant apps. Our TenantConfig feature is *just an option*. It's not standard in any sense, it's not even necessarily recommended for anything, it just exists and can be useful for some things. So I wouldn't try to make it consistent with TenantConfig since that's an unrelated thing. And it just happens to work a certain way.

Reply:
Right. I updated the array and the example https://github.com/archtechx/tenancy/commit/4016bc17ba75d89ad4e5344e7cbc31d8e238ee78